src/target/riscv/riscv-013.c

```diff
@@ -2237,10 +2237,18 @@ static int sample_memory_bus_v1(struct target *target,
 		if (result != ERROR_OK)
 			return result;
 
+		/* Discard the batch when we encounter a busy state. It's too
+		 * much hassle to try to recover partial data. We'll try again
+		 * with a larger delay. */
+		unsigned int sbcs_read_op = riscv_batch_get_dmi_read_op(batch, sbcs_key);
```

Comment:
Cosmetic thing: `sbcs_key` could be renamed to `sbcs_read_index` (or similar), if you'd like.

Reply:
Yes, that's better.